test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -1084,7 +1086,7 @@ object CheckAllAnswersAndSubmitControllerSpec {
     isUk: Boolean,
     isRebasing: Boolean,
     isATrust: Boolean
-  )(implicit messages: MessagesApi, lang: Lang): Unit = {
+  )(implicit messagesApi: MessagesApi, messages: Messages, lang: Lang): Unit = {
```

Comment:
do you an implicit `Messages` here now?

Reply:
That should have remained from the first commit.
Removed.